### DISCUSSION TITLE
get correct roundtime

### DISCUSSION THREAD

**Discussion by RaphaelHippe** [saul/demofile#222] [Questions] [answered]
Hello, me again. 

I am trying to get the roundtime of when e.g. a kill happened in during a round.

I am currently trying this: 

```javascript
    let stat = {
          attackerName: attacker.name,
          attackerHealth: attacker.health,
          weapon: e.weapon,
          // ...
          roundNumber: demoFile.gameRules.roundNumber,
          roundTime: demoFile.gameRules.getProp('DT_CSGameRules', 'm_iRoundTime')
        };
```

but that just returns 115 which I believe is the total round time. 

Thanks, 
Raphael Hippe

**Comment by saul** [comment, marked as answer]
Hi Raphael

You'll have to get the current demo time (e.g. `demo.currentTime`) when you see the `round_start` game event. Store that time in a variable, then when you see a kill, you can subtract the current `demo.currentTime` from the time you stored at round start. That will tell you how many seconds into the round the kill was.

Thanks
Saul
